tests/test_embedder.py

```diff
@@ -74,96 +74,74 @@ def check_dimension(
     bool
         True if all vectors have the correct dimension, False otherwise.
     """
-    for vector in vectors:
-        if is_sentence_embedding:
-            if len(vector) != dim:
-                return False
-        else:
-            for element in vector:
-                if len(element) != dim:
+    if is_sentence_embedding:
```

Comment:
the name of this isnt great. check that dimension... what? is correct? is small? is big? is present?